src/Agent.Worker/Handlers/NodeHandler.cs

```diff
@@ -523,5 +539,47 @@ private async Task<List<string>> ExecuteCommandAsync(IExecutionContext context,
 
             return outputs;
         }
+
+        private void PublishHandlerTelemetry(string realHandler)
+        {
+            var systemVersion = PlatformUtil.GetSystemVersion();
+            string expectedHandler = "";
+            if (Data is Node20_1HandlerData)
```

Comment:
nitpick: maybe it would be more elegant to use `switch` instead of multiple if-else

Reply:
updated